### ISSUE TITLE
Drop "type" attribute for JavaScript in HTML5

### ISSUE THREAD

**Issue by r2d02** [getgrav/grav#1837]
In HTML5 the attribute `type="text/javascript" ` is no longer needed/optional (it was needed until version 5 of HTML). I believe this verbosity is not needed and thus should be dropped.

FWIW it also gives a warning in the [W3 validator](https://validator.w3.org/)

[https://www.w3.org/TR/html5/document-metadata.html#processing-link-type](https://www.w3.org/TR/html5/document-metadata.html#processing-link-type)

**Comment by mahagr** [comment]
Yes, default type can be dropped -- though we need still to support `type=application/json"` and other types.

**Comment by pgr1973** [comment]
You can bodge it away with something like this in the twig-file(s):

`{{ assets.css()|replace({ 'type="text/css" ': ''}) }}`

`{{ assets.js()|replace({ 'type="text/javascript" ': ''}) }}`

`{{ assets.js('bottom')|replace({ 'type="text/javascript" ': ''}) }}`

For instance in the `/[THEME]/templates/partials/base.html.twig`
